modules/about-rhde.adoc

```diff
@@ -34,7 +34,7 @@ The latest release notes for each product that is part of {product-title} are av
 [id="device-edge-compatibility_{context}"]
 == {product-title} product release compatibility matrix
 
-{microshift-first} and {op-system} work together as a single solution for device-edge computing. To successfully pair your products, use the verified releases together for each as listed in the following table:
+{op-system-first} and {microshift} work together as a single solution for device-edge computing. Supported configurations of {product-title} use verified releases for each together as listed in the following table:
```

Comment:
```suggestion
{op-system-first} and {microshift} work together as a single solution for device-edge computing. You can update each component separately, but the versions must be compatible. An update of {microshift-short} from 4.14 to 4.16 requires a {op-system} update, for example. Supported configurations of {product-title} use verified releases for each together as listed in the following table:
```

Reply:
Something like this?

Reply:
I need to wait for the preview to be updated to see which did not yet happen....

Reply:
@aravipra ah sorry, I got confused. I thought the changes were already made and was waiting for the preview.
Yes, please go ahead and make the suggestes changes from Shauna so that I can see them in the preview. Thx!

Reply:
Done